packages/slider/src/SliderHandle.ts

```diff
@@ -0,0 +1,163 @@
+    },
+    toAttribute: (value: 'previous' | number): string => {
+        return value.toString();
+    },
+};
+
+const MaxConverter = {
+    fromAttribute: (value: string): number | 'next' => {
+        if (value === 'next') return value;
+        return parseFloat(value);
+    },
+    toAttribute: (value: 'next' | number): string => {
+        return value.toString();
+    },
+};
+
+export class SliderHandle extends Focusable {
+    public handleController?: Controller;
+
+    public get handleName(): string {
+        return this.name;
+    }
+
+    public get focusElement(): HTMLElement {
+        return this.handleController?.inputForHandle(this) ?? this;
+    }
+
+    @property({ type: Number })
+    value = 10;
+
+    @property({ type: Boolean, reflect: true })
+    public dragging = false;
+
+    @property({ type: Boolean })
+    public highlight = false;
+
+    @property({ type: String })
+    public name = '';
+
+    @property({ reflect: true, converter: MinConverter })
+    public min?: number | 'previous';
+
+    @property({ reflect: true, converter: MaxConverter })
+    public max?: number | 'next';
+
+    @property({ type: Number, reflect: true })
+    public step?: number;
+
+    @property({ type: Object, attribute: 'format-options' })
+    public formatOptions?: Intl.NumberFormatOptions;
+
+    @property({ attribute: false })
+    public getAriaHandleText: (
+        value: number,
+        numberFormat: Intl.NumberFormat
+    ) => string = (value, numberFormat) => {
+        return numberFormat.format(value);
+    };
+
+    protected updated(changedProperties: PropertyValues): void {
+        if (changedProperties.has('value')) {
+            const oldValue = changedProperties.get('value');
+            if (oldValue != null) {
+                this.handleController?.setValueFromHandle(this);
+            }
+        }
+        if (
+            changedProperties.has('numberFormat') ||
+            changedProperties.has('language')
+        ) {
+            delete this._numberFormatCache;
+        }
+        super.updated(changedProperties);
+    }
+
+    @property({ attribute: false })
+    public normalization: SliderNormalization = defaultNormalization;
+
+    public dispatchInputEvent(): void {
+        if (!this.dragging) {
+            return;
+        }
+        const inputEvent = new Event('input', {
+            bubbles: true,
+            composed: true,
+        });
+
+        this.dispatchEvent(inputEvent);
+    }
+
+    protected _numberFormatCache:
+        | { numberFormat: Intl.NumberFormat; language: string }
+        | undefined;
+    protected getNumberFormat(): Intl.NumberFormat {
+        const language = this.handleController?.language ?? navigator.language;
+        if (
+            !this._numberFormatCache ||
+            language !== this._numberFormatCache.language
+        ) {
+            this._numberFormatCache = {
+                language,
+                numberFormat: new Intl.NumberFormat(
```

Comment:
We use https://www.npmjs.com/package/@internationalized/number to better align Number Field to other Adobe offerings, should we do the same here?

Reply:
Done.